starboard/CHANGELOG.md

```diff
@@ -5,205 +5,230 @@ This file will be updated each time a new Starboard version is released.
 Each section in this file describes the changes made to the Starboard interface
 since the version previous to it.
 
-**NOTE: Starboard versions 9 and below are no longer supported.**
+**NOTE: Starboard versions 13 and older are no longer supported.**
 
 ## Version 16
+A main change in starboard version 16 is add/migrate to POSIX APIs.
```

Comment:
Capitalize `Starboard` here

Reply:
`A key update in Starboard version 16 is the adoption of POSIX APIs.` is used